highlighting/rules_automaton.py

```diff
@@ -150,8 +150,8 @@ def rule_level3():
         'regex': START_WORD + K("at") ,
```

Comment:
Can we call this K() function something more descriptive like `translate` or `get_translated_keyword`?

Reply:
I can do it, but it should be a short name so as not to make the lines too long (which will reduce readability)

I'll try with `translate`

Reply:
it's done !